engine/apps/alerts/models/alert_receive_channel.py

```diff
@@ -527,8 +527,8 @@ def heartbeat_module(self):
         return getattr(heartbeat, self.INTEGRATIONS_TO_REVERSE_URL_MAP[self.integration], None)
 
     # Demo alerts
-    def send_demo_alert(self, force_route_id=None, payload=None):
```

Comment:
`force_route_id` seemed to _only_ be passed in when this method was called from `ChannelFilter. send_demo_alert` (which was already deprecated and is now removed in this PR)